components/automate-ui/src/app/components/sidebar-entry/sidebar-entry.component.html

```diff
@@ -12,6 +12,7 @@
 <ng-template #sidebarEntryTemplate>
   <chef-icon class="sidebar-icon" *ngIf="icon"
     [style.transform]="'rotate(' + (iconRotation ? iconRotation : 0) + 'deg)'">{{icon}}</chef-icon>
+  <div *ngIf="customIcon === 'chef-servers'" class="chef-servers-icon"></div>    
```

Comment:
Would prefer if this class did not need to know about its callers (that goes against the Open/Closed Principle -- the "O" in [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design))). Perhaps we could just do something like this:
```suggestion
  <div *ngIf="customIcon" [attr.class]="customIcon"></div>    
```

Reply:
@msorens nice catch :1st_place_medal: , incorporated :+1: